tests/validation/tests/v3_api/test_oke_cluster.py

```diff
@@ -0,0 +1,203 @@
+import pytest
+from .common import *  # NOQA
+
+OKE_TENANCY_OCID = os.environ.get('RANCHER_OKE_TENANCY_OCID', "")
+OKE_COMPARTMENT_OCID = os.environ.get('RANCHER_OKE_COMPARTMENT_OCID', "")
+OKE_USER_OCID = os.environ.get('RANCHER_OKE_USER_OCID', "")
+OKE_FINGERPRINT = os.environ.get('RANCHER_OKE_FINGERPRINT', "")
+OKE_PRIVATE_KEY_PATH = os.environ.get('RANCHER_OKE_PRIVATE_KEY_PATH', "")
+OKE_PRIVATE_KEY_PASSPHRASE = os.environ.get('RANCHER_OKE_PRIVATE_KEY_PASSPHRASE', "")
+OKE_REGION = os.environ.get('RANCHER_OKE_REGION', None)
+OKE_VERSION = os.environ.get('RANCHER_OKE_VERSION', None)
+OKE_NODE_SHAPE = os.environ.get('RANCHER_OKE_NODE_SHAPE', None)
+OKE_NODE_IMAGE = os.environ.get('RANCHER_OKE_NODE_IMAGE', None)
+
+okecredential = pytest.mark.skipif(not (
+        OKE_TENANCY_OCID and OKE_COMPARTMENT_OCID and OKE_USER_OCID and OKE_FINGERPRINT and OKE_PRIVATE_KEY_PATH),
+                                   reason='OKE Credentials not provided, '
+                                          'cannot create cluster')
+
+
+def test_error_get_oke_latest_versions_missing_key():
+    oci_cred_body = {
+        "tenancyOCID": "ocid1.tenancy.oc1..aaaaaa",
+        "userOCID": "ocid1.user.oc1..aaaaaaaa",
+        "region": "us-ashburn-1",
+        "fingerprint": "xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx",
+    }
```

Comment:
So the values can be any random values as long as they follow the pattern? 

Why does this one not have the `privateKey` item comparing with those in the other two methods?

Reply:
It's missing the private key because it's a negative/error scenario test `test_error_get_oke_latest_versions_missing_key`.